pkg/net/net.go

```diff
@@ -20,13 +20,17 @@ type NICCapability struct {
 	CanEnable bool   `json:"can_enable"`
 }
 
+type NICAddress struct {
+	PCI string `json:"pci"`
+	// TODO(fromani): add other hw addresses (USB) when we support them
```

Comment:
So this would essentially be a union type, yes? There would be a set of different address fields, and only one would be expected to be set?

Reply:
Yes, that's the intent. The reason is I'm aware of pci-connected (desktop, servers...) and usb-connected (laptops, raspberry pi) network interfaces. So that's at least two.
I'm not sure what's the best way to model this facet and I'm very open to suggestions.